tests/test_web_functional.py

```diff
@@ -1469,7 +1469,7 @@ def make_handler(appname):
     assert 413 == resp.status
     resp_text = await resp.text()
     assert 'Maximum request body size 2097152 exceeded, ' \
-           'actual body size 2097166' in resp_text
+           'actual body size' in resp_text
```

Comment:
If the number is different, we need some explanation to comment here why this is happening. Is it possible to figure out the precise number that would appear here?

Reply:
see my comment https://github.com/aio-libs/aiohttp/pull/4463#discussion_r360393786

Reply:
Updated tests, WDYT @webknjaz ?